vividus/src/main/java/org/vividus/results/ResultsProvider.java

```diff
@@ -14,11 +14,22 @@
  * limitations under the License.
  */
 
-package org.vividus;
+package org.vividus.results;
 
+import java.util.List;
+import java.util.Map;
 import java.util.Optional;
 
-public interface IRunStatusProvider
+import org.vividus.report.allure.model.Status;
+import org.vividus.results.model.Failure;
+import org.vividus.results.model.NodeType;
+import org.vividus.results.model.Statistic;
+
+public interface ResultsProvider
```

Comment:
Results sounds too abstract, results of scenario, step, story

Reply:
please propose another name if you have any on your mind
in general it should be generic, since it provides:
- test execution status
- test execution statistics
- list of happened failures